recipes/gaia/build.sh

```diff
@@ -0,0 +1,16 @@
+
+# From the gaia readme:
+#
+# https://github.com/OpenDataAnalytics/gaia/blob/master/README.md#installation
+#
+# # Installation issues - Ubuntu:
+# You may run into cpl_port.h No such file or directory. Do the following:
+# sudo apt-get install libgdal-dev
+# export CPLUS_INCLUDE_PATH=/usr/include/gdal
+# export C_INCLUDE_PATH=/usr/include/gdal
+#
```

Comment:
I would remove these comments as they are not relevant. We use gdal from conda-forge.

Reply:
These comments explain the presence of the the environment variable creation just below.

Reply:
By the way, they're just pasted from that link in the comment section (the gaia README).